src/rpc/rawtransaction.cpp

```diff
@@ -35,6 +35,11 @@
 
 using namespace std;
 
+#ifdef ENABLE_WALLET
+std::string HelpRequiringPassphrase(CWallet *);
```

Comment:
I'd kinda prefer we add another declaration outside of a header in an unrelated file...can we keep it in server.h and ifdef ENABLE_WALLET it?